test/unit/org/apache/cassandra/db/guardrails/ThresholdTester.java

```diff
@@ -118,15 +118,41 @@ protected void testValidationOfThresholdProperties(String warnName, String failN
                   .hasMessageContaining(guardrail.name + "_warn_threshold should be lower than the fail threshold");
     }
 
-    protected void assertThresholdValid(String query) throws Throwable
+    protected void assertMaxThresholdValid(String query) throws Throwable
```

Comment:
We might make this class and this method abstract, and have new `MinThreshodTester` and `MaxThresholdTester` subclasses implementing the method. Not sure it's worth the fuss, though.